File: 12-count-inversions/python/main.py
import json
import sys


def solve(arr):
    # TODO: return the number of inversions (pairs i < j with arr[i] > arr[j])
    pass


def main():
    obj = json.load(sys.stdin)
    print(solve(obj["arr"]))


if __name__ == "__main__":
    main()

File: 12-count-inversions/go/main.go
package main

import (
	"encoding/json"
	"fmt"
	"os"
)

type input struct {
	N   int   `json:"n"`
	Arr []int `json:"arr"`
}

func solve(arr []int) int64 {
	// TODO: return the number of inversions (pairs i < j with arr[i] > arr[j])
	_ = arr
	return 0
}

func main() {
	var in input
	if err := json.NewDecoder(os.Stdin).Decode(&in); err != nil {
		fmt.Fprintln(os.Stderr, err)
		os.Exit(1)
	}
	fmt.Println(solve(in.Arr))
}
